cleanlab/classification.py

```diff
@@ -73,7 +74,7 @@ def score(self, X, y, sample_weight = None):
 Note
 ----
 
-* `s` - denotes *noisy labels*. This is just dataset labels, maybe with errors.
+* `labels` - denotes *noisy labels*. This is just dataset labels, maybe with errors.
```

Comment:
"dataset labels"
-> 
"given labels in the original dataset"

Reply:
done